Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
```

Comment:
Do these files have to be in the root directory? Is it possible to move them elsewhere?